### DISCUSSION TITLE
Why Api reference page link not working for me?

### DISCUSSION THREAD

**Discussion by kindacoder** [fattureincloud/api#389] [Q&A] [answered]
Hey folks!
None of the links working for me on page.  [https://developers.fattureincloud.it/api-reference](https://developers.fattureincloud.it/api-reference)
The page keeps loading for me.

Example:

[https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/info/payment_methods](https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/info/payment_methods)

<img width="1507" alt="Screenshot 2023-12-24 at 12 01 48 PM" src="https://github.com/fattureincloud/api/assets/28186433/75d1e065-e821-44f1-a770-a52517b14701">

**Comment by valmoz** [comment, marked as answer]
Hi @kindacoder 
The problem is related to your location.
Our API reference is based on [RapiDoc](https://rapidocweb.com/), a tool that uploads our OpenAPI file and uses it to let you test our methods. We published our OpenAPI file from Github, and unfortunately, it seems that the Github link is accessible only from Europe. I think that Github mirrors the same file on different servers, so the one we used is unreachable from your zone.

Here you can find some solutions to your problem, unfortunately, I cannot test them because I cannot replicate the issue:
- When RapiDoc fails to load the OpenAPI, it should show a textbox where you can insert a link to an OpenAPI file. If that happens, you should be able to insert the link of the [openapi-enriched.yaml](https://github.com/fattureincloud/openapi-fattureincloud/blob/master/openapi-enriched.yaml) file (you must use the link of the Raw version)
- If you cannot do it, then we suggest using our [Postman Collection](https://developers.fattureincloud.it/docs/quickstarts/postman-collection)
- If even this solution doesn't work, then I think it would be faster to use one of our SDKs to test our APIs directly from code

Let us know if one of these solutions worked for you.

Thanks
Mauro

**Comment by kindacoder** [comment]
It's working now.
Thanks @valmoz

**Reply by valmoz** [reply]
@kindacoder you're welcome
